prometheus/src/cluster_metrics.rs

```diff
@@ -52,5 +81,60 @@ pub fn write_cluster_metrics<W: io::Write>(
         },
     )?;
 
+    // Vote accounts information
+    for vote_account in vote_accounts.iter() {
+        write_metric(
+            out,
+            &MetricFamily {
+                name: "solana_node_last_vote_slot",
+                help:
+                    "The voted-on slot of the validator's last vote that got included in the chain",
+                type_: "gauge",
+                metrics: banks_with_commitments.for_each_commitment(|bank| {
+                    let vote_info = get_vote_state(bank, vote_account)?;
+                    Some(
+                        Metric::new(vote_info.last_vote)
+                            .with_label("identity_account", identity_pubkey.to_string())
+                            .with_label("vote_account", vote_account.to_string()),
+                    )
+                }),
+            },
+        )?;
+
+        write_metric(
+            out,
+            &MetricFamily {
+                name: "solana_node_vote_balance_sol",
+                help: "The current node's vote account balance",
+                type_: "gauge",
+                metrics: banks_with_commitments.for_each_commitment(|bank| {
+                    let vote_info = get_vote_state(bank, vote_account)?;
+                    Some(
+                        Metric::new_sol(vote_info.balance)
+                            .with_label("identity_account", identity_pubkey.to_string())
+                            .with_label("vote_account", vote_account.to_string()),
+                    )
+                }),
+            },
+        )?;
+
+        write_metric(
+            out,
+            &MetricFamily {
+                name: "solana_node_vote_credits",
+                help: "The current node's vote vote credits for current epoch",
```

Comment:
```suggestion
                help: "The total number of vote credits credited to this vote account",
```
Also here, this is no longer specific to this node, and also the credits are not specific to the current epoch, they are the total since the beginning of time. I see that the `VoteState` representation changed to also track it since the beginning of the epoch, but this value is still the vote credits since the beginning of time.

(And that is also more useful to us: once Prometheus has a time series of this, we can compute the number of vote credits gained between any two points in time, not just the start of the epoch.)

Reply:
So this (https://github.com/ChorusOne/solana/blob/50604103de3c90f8a5aac135bfe13b0a7ccd614f/programs/vote/src/vote_state/mod.rs#L1047)
is cumulative?
The comment on `VoteState` says:
```rust
/// history of how many credits earned by the end of each epoch
/// each tuple is (Epoch, credits, prev_credits)
```
I think I got confused here. Testing it with `solana-test-validator` showed me that this is indeed accumulating 👍

Reply:
Thanks for confirming!